src/Toolchain/ToolchainEnv.ts

```diff
@@ -113,11 +114,20 @@ class ToolchainEnv extends Env {
   }
 
   listInstalled(): Toolchain[] {
-    return this.compiler.getToolchainTypes()
-        .map((type) => this.compiler.getInstalledToolchains(type))
-        .reduce((r, a) => {
-          return r.concat(a);
-        });
+    const toolchains = this.compiler.getToolchainTypes()
+                           .map((type) => this.compiler.getInstalledToolchains(type))
+                           .reduce((r, a) => {
+                             return r.concat(a);
+                           });
+    if (toolchains.length > 0) {
+      // TODO(jyoung) What item will be default toolchain?
+      this.toolchain = toolchains[0];
+    }
```

Comment:
I think "setting default toolchain" is not related with "getting installed toolchain".
It would be better to separate this later :)

Reply:
I think new installed toolchain will be the default toolchain. Because user installed this toolchain to use it. 

And there is no way to set the default toolchain as of now. So this is the only way to use this toolchain when user request to compile.